src/nanoemoji/colors.py

```diff
@@ -229,3 +229,16 @@ def to_ufo_color(self):
 
     def opaque(self):
         return self._replace(alpha=1.0)
+
+    def to_string(self):
```

Comment:
this could even be the magic `__str__` method if you like, so str(color) or print(color) just work

Reply:
I waffled ... I kind of like the tuple print